conda_smithy/configure_feedstock.py

```diff
@@ -1565,10 +1569,24 @@ def render_README(jinja_env, forge_config, forge_dir, render_info=None):
                 variant_name, _ = os.path.splitext(filename)
                 variants.append(variant_name)
 
+    subpackages_metas = OrderedDict((meta.name(), meta) for meta in metas)
+    subpackages_about = []
+    for name, m in subpackages_metas.items():
+        # use the top-level (feedstock) about entries if the subpackages' are undefined
```

Comment:
I do notice this is not the current behavior when conda-build generates metadata for each subpackage. Should I instead follow conda-build behavior?

Reply:
Following conda-build is preferred. Also, when there's only a top level about, there's no need to copy. Just one about with `package_name`.

Reply:
@isuruf do you mean if there's only a top level about, we do this instead?
```
About output1, output2, output3, ...
-----------------------------------

License: ...

Summary: ...

```
Because in this case conda-build would copy the top-level about to all sub packages (hence the deviating behavior). I do like this, but I am not sure if this is what you meant.

Reply:
ah I misread, so you just want this?
```
About package_name (name of top-level package)
--------
```

Reply:
I took a stab and made the change in commit 8e992bf, let me know if it's better, thx!